components/fibery/sources/entity-created/entity-created.mjs

```diff
@@ -0,0 +1,21 @@
+import base from "../common/webhooks.mjs";
+
+export default {
+  ...base,
+  key: "fibery-entity-created",
+  name: "New Entity Created",
+  description: "Emit new event for every created entity of a certain type",
+  type: "source",
+  version: "0.0.1",
```

Comment:
why not include explicit dedupe strategy?

Reply:
Ahh right, I was emitting event for 1 whole sequence, which didn't have just one `id`. Then I changed that but forgot to update the code to emit the `id`s.